Kickstarter-iOS/Views/Controllers/SurveyResponseViewController.swift

```diff
@@ -60,7 +61,15 @@ internal final class SurveyResponseViewController: DeprecatedWebViewController {
     self.viewModel.outputs.webViewLoadRequest
       .observeForControllerAction()
       .observeValues { [weak self] request in
-        self?.webView.loadRequest(request)
+        self?.webView.load(request)
+      }
+
+    self.viewModel.outputs.extractFormDataWithJavaScript
+      .observeForUI()
+      .observeValues { [weak self] js in
+        self?.webView.evaluateJavaScript(js) { result, _ in
+          self?.viewModel.inputs.didEvaluateJavaScriptWithResult(result)
```

Comment:
We run a small bit of JS in the `webView` and call the `viewModel` with the result.